train/uft/tuning.py

```diff
@@ -123,7 +122,7 @@ def objective (trial):
 
 def run_tuning():
     study = optuna.create_study(direction="maximize", study_name="uft-tuning")
-    study.optimize(objective, n_trials=5)
+    study.optimize(objective, n_trials=15)
```

Comment:
i think n_trials is too small? how much time did this take? it depends on the time but i think we want at least 20 trials.

Reply:
i can try fixing epochs to 17 and then adding trials to 25.

Reply:
thanks!